com.oracle.truffle.r.runtime/src/com/oracle/truffle/r/runtime/ffi/VectorRFFIWrapper.java

```diff
@@ -524,6 +570,43 @@ protected static Object doComplexVector(RComplexVector vector, int index, int va
             return vector;
         }
 
+        // TODO: Add other specializations
+        @Specialization(limit = "1")
+        protected static Object doAltIntegerVector(RAltIntVectorData vector, int index, int value,
```

Comment:
Tohle je (opet) priklad stareho provolani se do altrep nativnich metod. V budoucnu to chci nahradit nekterym z nodu v `AltrepRFFI` resp. `AltrepDownCallNode`.